### PR TITLE
Warn our users about the forklift

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,11 @@ Catello is backed by candlepin that our users love:
 ============================
 ![Candlepin](cattlepin.jpg)
 
+A word of warning:
+==================
+![Use No Fork Lifts](no_forklifts.jpg)
+![Prohibit Use Of Forklift](prohibit_forklift.jpg)
```
Comment (ehelms) [inline README.md]: Should users be encouraged to [catlift](https://github.com/Catello/catlift) only?

Comment (mdellweg) [reply README.md]: I guess so.

**Comment by waldenraines** [comment]
> ehelms merged commit 6460600 into Catello:master  19 seconds ago

I thought we weren't accepting PRs and you were supposed to force push to master instead.  @daviddavis?

**Comment by mdellweg** [comment]
Well, you can always redo the merge fast-forward and force push it. That is, what force pushes are really good at. ;)

**Comment by jlsherrill** [comment]
We aren't accepting PRs and suggest force pushing yes.  However so far we have refused to document this.

While i support this policy, i will not support documenting it and will NACK any Pull Request that suggests we document the fact we don't accept Pull Requests. 

@mdellweg you now how commit access! Congrats!

**Comment by CalvinHartwell** [comment]
:joy_cat: :joy_cat: :joy_cat: :joy_cat:

**Pull request merged**